crates/cast/tests/e2e/account/create.rs

```diff
@@ -91,9 +91,9 @@ pub async fn test_happy_case_generate_salt() {
 #[tokio::test]
 pub async fn test_happy_case_add_profile() {
     let current_dir = Utf8PathBuf::from(duplicate_directory_with_salt(
-        CONTRACTS_DIR.to_string() + "/v1/balance",
+        CONTRACTS_DIR.to_string() + "/map",
         "put",
-        "1",
+        "10",
```

Comment:
Hmm, why is it changed? 🤔

Reply:
two weeks passed, and I don't remember 😅 my guess is a few more tests used this account and something failed because of too few funds, but not sure. If you want, I can try anc check it, but I don't think this is that relevant tbh :+1: